plugins/blip/basics/logic/actions.js

```diff
@@ -67,18 +67,11 @@ basicsActions.setSiteChangeEvent = function(sectionName, selectedKey, selectedLa
 
   metricsFunc('Selected ' + selectedLabel);
 
-  var newProfile = _.merge(this.app.props.patient.profile, {
-    patient: {
-      settings: {
-        siteChangeSource: selectedKey,
-      },
-    },
+  var newSettings = _.merge(this.app.props.patient.settings, {
```

Comment:
I missed this the first time around (and I guess it first came in via #303), but now that I'm hunting down places where we were mistakenly mutating things in our redux state tree, I found this too.

All of this state management strategy via binding in a component (`this.app`) is pre-redux migration and definitely not how we want to keep moving forward, but regardless, we [cannot mutate component props](https://facebook.github.io/react/docs/components-and-props.html#props-are-read-only).

I'll be putting together a fix for this as part of the effort to unblock things. CC @jh-bate @hntrdglss @krystophv